### PR TITLE
block[weather]: Fix nws apparent temperature

### PR DESCRIPTION
**Pull request by bim9262**
australian_apparent_temp requires m/s not km/h.
convert temperature back to fahrenheit if original temperature is in fahrenheit

@Cognoscan does this capture the changes that you mentioned on your PR?

### CODE REVIEW

**Comment by Cognoscan** [comment]
Almost. There's one other possible issue, which is that other services in using metric units use m/s for wind, whereas NWS will return km/h when metric units are requested. I'm not sure if that matters or not.

**Comment by bim9262** [comment]
Good call, I think it would be best to keep the speeds in standard units regardless of backend

File: src/blocks/weather/nws.rs [hunk 1]
```diff
@@ -177,8 +185,13 @@ impl ApiForecast {
             (self.temperature.value - 32.0) * 5.0 / 9.0
         };
         let humidity = self.relative_humidity.value;
-        let wind_speed = self.wind_kmh();
-        australian_apparent_temp(temp, humidity, wind_speed)
+        let wind_speed = self.wind_kmh() / 3.6;
```
Comment (MaxVerevkin) [inline src/blocks/weather/nws.rs:188]: Isn't this just `= self.wind_speed();`?

Comment (bim9262) [reply src/blocks/weather/nws.rs:188]: Wind speed could be mph or m/s

Comment (MaxVerevkin) [reply src/blocks/weather/nws.rs:188]: But wind_speed() is the function which you've added which always returns m/s?

**Pull request merged**